tests/integration/ha/test_storage.py

```diff
@@ -113,6 +112,7 @@ async def test_storage_reuse_after_scale_down(
 
 @pytest.mark.group(1)
 @pytest.mark.abort_on_fail
+@pytest.mark.skip(reason="This test does not work currently, need to clarify the functionality.")
```

Comment:
The goal here is essentially to destroy a cluster while keeping the storage, then to deploy a new cluster with the previously kept storage from the previous cluster, and then see if the data of the continuous writes has restored.